### PR TITLE
feat: convert to Eleventy (resolves #41)

### LINKED ISSUE #41: Convert to Eleventy via fluidic-11ty
## Is your feature request related to a problem?

As a prerequisite for further updates, this website should be converted from Docpad to Eleventy.

## Describe the solution you'd like

Rearchitect the site using [fluidic-11ty](https://github.com/fluid-project/fluidic-11ty).

## Describe alternative solutions you've considered

Not applicable.

## Additional context or notes

Not applicable.

### PR DESCRIPTION
**Pull request by greatislander**
This PR converts the Inclusive Design Guide to use [Eleventy](https://11ty.dev), based on the [Fluidic Eleventy](https://github.com/fluid-project/fluidic-11ty) starter kit.

### Deployment

This PR configures a Dockerfile for Eleventy deployment (based on the [fluidproject.org](fluidproject.org) and [floeproject.org](floeproject.org) Eleventy sites).

### Markup

All HTML in Markdown files has been replaced with standard Markdown syntax, with three exceptions.

* Lists with custom SVG bullets inlined via [Nunjucks `includes`](https://mozilla.github.io/nunjucks/templating.html#include) from [`src/_includes/svg/`](src/_includes/svg)
* Images wrapped in `<figure>` / `<figcaption>` tags
* Sectioning elements on the [home page](https://github.com/inclusive-design/guide.inclusivedesign.ca/blob/9bf91fe0da41a8bbff64e66898e5d7dde7ab78fc/src/index.md#L10)

These changes necessitate some post-processing of the markup using an Eleventy transform to accomplish the following tasks:

* Add relevant classes to bulleted lists to properly style the SVG bullets: <https://github.com/inclusive-design/guide.inclusivedesign.ca/blob/37d701cc91cf5095627457212496ab1dadf68dc2/src/transforms/parse-transform.js#L35-L45>
* Process Markdown within `<figure>` / `<figcaption>` tags: <https://github.com/inclusive-design/guide.inclusivedesign.ca/blob/37d701cc91cf5095627457212496ab1dadf68dc2/src/transforms/parse-transform.js#L47-L58>
* Process Markdown within sectioning elements on the home page: <https://github.com/inclusive-design/guide.inclusivedesign.ca/blob/37d701cc91cf5095627457212496ab1dadf68dc2/src/transforms/parse-transform.js#L59-L63>

### Redirects

Redirects via [`docpad-plugin-redirector`](https://www.npmjs.com/packages/docpad-plugin-redirector) have been moved to the load balancer (see #43).

### Site Structure

The old Docpad site-structure.json file has been replaced by a menu and sidebar navigation implementation that uses the [Eleventy Navigation](https://www.11ty.dev/docs/plugins/navigation/) plugin.

## Related Issues

Resolves #41.

### CODE REVIEW

**Review by gtirloni — APPROVED** [review]

File: src/documents/activities/FunctionalMashup.md [hunk 1]
```diff
@@ -31,8 +37,8 @@ If desired and available you can provide additional tools and materials that can
...
 ## Why
+
 The goal of this activity is to promote creative thinking over immediate feasibility. It encourages participants to dig deeper to find unexpected and creative ways of addressing a problem.
-<ul class="idg-articleContentUseWhyHow">
-<li><span role="presentation" class="idg-iconPractices">{{{getCategoryIcon "practices"}}}</span> [Practice Co-Design](/practices/PracticeCoDesign.html)</li>
-<li><span role="presentation" class="idg-iconPractices">{{{getCategoryIcon "practices"}}}</span> [Collaborate](/practices/Collaborate.html)</li>
-</ul>
+
+* {% include 'svg/icon-practices.svg' %} [Practice Co-Design](/practices/PracticeCoDesign.html)
+* {% include 'svg/icon-practices.svg' %} [Collaborate](/practices/Collaborate.html)
```
Comment (jhung) [inline src/documents/activities/FunctionalMashup.md:44]: Not sure why, but the default bullets are appearing for this list only.

image

Comment (greatislander) [reply src/documents/activities/FunctionalMashup.md:44]: Weird! I'll investigate.

Comment (greatislander) [reply src/documents/activities/FunctionalMashup.md:44]: Resolved in 719f242.

**Comment by gtirloni** [comment]
@greatislander I added some netlify redirects to `src/static/_redirects`. Could rebase `main` into here? Sorry for the trouble.

**Comment by gtirloni** [comment]
I think you'll need a netlify.toml file to override the build command in the configuration so it builds with 11ty:

```
[build]
  command = "npm run build"
  publish = "dist"
```

Right now it's looking for docpad.

**Review by jhung — APPROVED** [review]

**Pull request merged**